applications/osb-portal/src/components/workspace/drawer/WorkspaceDrawer.tsx

```diff
@@ -124,7 +125,7 @@ export const WorkspaceDrawer: React.FunctionComponent<WorkspaceDrawerProps> = ({
         }}
       >
         <div className={`${open ? classes.drawerContent : ''} verticalFit`}>
-          <WorkspaceInteractions open={open} openResource={setCurrentResource} />
+          <WorkspaceInteractions open={open} openResource={setCurrentResource} cloneWorkspace={cloneWorkspace}/>
```

Comment:
The cloneWorkspace action would be better passed implicitly by the redux connector as other callbacks are here. If you follow the path from index.ts you will find the redux connector logic and you'll also see tha WorkspaceActions are passed at the end of the day to the WorkspaceInteractions component, so no need for propagating properties